### PR TITLE
Feature/307

### PR DESCRIPTION
**Pull request by marcoAntonioNina**
Add api User profile

api/1.0/admin/profile GET/PUT

issue https://github.com/ProcessMaker/bpm/issues/307

### CODE REVIEW

File: routes/api.php [hunk 1]
```diff
@@ -31,6 +31,10 @@
             Router::get('users', 'Administration\UsersController@index');
             Router::get('users/{user}', 'Administration\UsersController@get');
 
+            //User profile
+            Router::get('admin/{user}/profile', 'Administration\UsersController@profile');
+            Router::put('admin/{user}/profile', 'Administration\UsersController@update');
+
```
Comment (velkymx) [inline routes/api.php]: The logged in use should be able to directly poll the api end point without an id. 

ie: api/1.0/admin/profile

These other routes should be the same. This is different from the admin/user end points.

Comment (marcoAntonioNina) [reply routes/api.php]: Changed

File: ProcessMaker/Http/Controllers/Api/Administration/UsersController.php [hunk 2]
```diff
@@ -2,8 +2,13 @@
```
Comment (velkymx) [inline ProcessMaker/Http/Controllers/Api/Administration/UsersController.php]: I'd like to move this controller to User/ProfileController there will be an Admin/UserController for the admin area.

File: ProcessMaker/Http/Controllers/Api/Administration/UsersController.php [hunk 3]
```diff
@@ -74,4 +79,82 @@ public function get(Request $request, User $user)
...
+    /**
+     * Load profile user
+     *
+     * @return JsonResponse
+     */
+    public function profile()
+    {
+        $url = '';
+        $user = User::find(Auth::id());
+        if (!empty($user->avatar) && Storage::disk('profile')->exists($user->avatar)) {
```
Comment (velkymx) [inline ProcessMaker/Http/Controllers/Api/Administration/UsersController.php]: How do we retrieve the user's avatar image? Is there an image url?

**Pull request merged**